aws-athena/ctas/location-census_2010.sql

```diff
@@ -0,0 +1,91 @@
+{{
+    config(
+        materialized='table',
+        partitioned_by=['year'],
+        bucketed_by=['pin10'],
+        bucket_count=1
+    )
+}}
+
+WITH distinct_pins AS (
+    SELECT DISTINCT
+        x_3435,
+        y_3435
+    FROM {{ source('spatial', 'parcel') }}
+),
+
+distinct_joined AS (
+    SELECT
+        dp.x_3435,
+        dp.y_3435,
+        MAX(CASE
+            WHEN cen.geography = 'block_group' THEN cen.geoid
+        END) AS census_block_group_geoid,
+        MAX(CASE
+            WHEN cen.geography = 'block' THEN cen.geoid
+        END) AS census_block_geoid,
+        MAX(CASE
+            WHEN cen.geography = 'congressional_district' THEN cen.geoid
+        END) AS census_congressional_district_geoid,
+        MAX(CASE
+            WHEN cen.geography = 'county_subdivision' THEN cen.geoid
+        END) AS census_county_subdivision_geoid,
+        MAX(CASE
+            WHEN cen.geography = 'place' THEN cen.geoid
+        END) AS census_place_geoid,
+        MAX(CASE
+            WHEN cen.geography = 'puma' THEN cen.geoid
+        END) AS census_puma_geoid,
+        MAX(CASE
+            WHEN cen.geography = 'school_district_elementary' THEN cen.geoid
+        END) AS census_school_district_elementary_geoid,
+        MAX(CASE
+            WHEN cen.geography = 'school_district_secondary' THEN cen.geoid
+        END) AS census_school_district_secondary_geoid,
+        MAX(CASE
+            WHEN cen.geography = 'school_district_unified' THEN cen.geoid
+        END) AS census_school_district_unified_geoid,
+        MAX(CASE
+            WHEN cen.geography = 'state_representative' THEN cen.geoid
+        END) AS census_state_representative_geoid,
+        MAX(CASE
+            WHEN cen.geography = 'state_senate' THEN cen.geoid
+        END) AS census_state_senate_geoid,
+        MAX(CASE
+            WHEN cen.geography = 'tract' THEN cen.geoid
+        END) AS census_tract_geoid,
+        MAX(CASE
+            WHEN cen.geography = 'zcta' THEN cen.geoid
+        END) AS census_zcta_geoid,
+        cen.year
+    FROM distinct_pins AS dp
+    LEFT JOIN {{ source('spatial', 'census') }} AS cen
+        ON ST_WITHIN(
+            ST_POINT(dp.x_3435, dp.y_3435),
+            ST_GEOMFROMBINARY(cen.geometry_3435)
+        )
+    WHERE cen.year = '2021'
```

Comment:
**suggestion:** We should probably make this the actual 2010 data year:

```suggestion
    WHERE cen.year = '2010'
```

Reply:
2012 is the earliest we have. Should suffice.